README.adoc

```diff
@@ -8,36 +8,16 @@
-
-It is very easy to analyze a C, C++ and Objective-C project on SonarCloud with code coverage https://docs.sonarcloud.io/getting-started/azure-devops/[using Azure DevOps]:
-
-. Create a `sonar-project.properties` file to store your configuration
-. Install SonarCloud extension for your organization:
-.. Open Organization settings (in the bottom left corner of organization view)
-.. Open "Extensions" page (in the General category)
-.. Press "Browse marketplace" and find "SonarCloud"
-.. Select your organization and click "Install"
-. Add the SonarCloud connection to your project:
-.. Open Project settings (in the bottom left corner)
-.. Open "Service connections" page (in the Pipelines category)
-.. Press "New connection" and select the "SonarCloud"
-.. Fill in the `SONAR_TOKEN`
-.. Name your connection `SonarCloud` (to reference it later in `azure-pipelines.yml`)
-. In your `azure-pipelines.yml` file:
-.. Add the `SonarCloudPrepare` task and configure it:
-* Specify the `SonarCloud Service Endpoint` as `SonarCloud` - the connection you created earlier
-* Choose your `organization`
-* Choose "Use standalone scanner" (`scannerMode: 'CLI'`)
-* Choose "Store configuration with my source code (sonar-project.properties)" (`configMode: 'file'`)
-* Initialize the extra property `sonar.cfamily.vscoveragexml.reportsPath` with the path that will contain your XML coverage report
-.. Add a task to download the https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-build-wrapper[Build Wrapper]
-.. Wrap your compilation with the Build Wrapper
-.. Add a `VSTest` task to launch your tests. Be sure to enable code coverage and to convert the result file into an XML file
-.. Add the `SonarCloudAnalyze` task
+*The test coverage shown here works on both SonarQube and SonarCloud. This project is analysed on https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_windows-vscoverage-msbuild-azure-sc[SonarCloud]!*
+
+NOTE: This simple example aims at demonstrating code coverage in action. It explains how to add coverage on a project for which the analysis is already configured. Some analysis https://github.com/orgs/sonarsource-cfamily-examples/repositories?q=topic%3Aconfiguration+msbuild-azure&type=all&language=&sort=[configuration examples] are available. +
+Adding code coverage is done the same way regardless of whether you use the Build Wrapper or a Compilation Database to configure the analysis.
+
+To get coverage and feed it to the sonar-scanner, use the following procedure.
+
+. In your azure-pipelines.yml file, Initialize the extra property `sonar.cfamily.vscoveragexml.reportsPath` with the path that will contain your XML coverage report
+. Add a `VSTest` task to launch your tests. Be sure to enable code coverage and to convert the result file into an XML file
+. Add the `SonarCloudAnalyze` task
```

Comment:
```suggestion
. Make sure the `SonarCloudAnalyze` task runs after the `VSTest` task
```
If we assume the analysis is configured, `SonarCloudAnalyze` task is already there.